pkg/util/clusteragent/clusteragent.go

```diff
@@ -106,6 +109,8 @@ func (c *DCAClient) init() error {
 
 	c.clusterAgentAPIRequestHeaders = http.Header{}
 	c.clusterAgentAPIRequestHeaders.Set(authorizationHeaderKey, fmt.Sprintf("Bearer %s", authToken))
+	podIP := getLocalIP()
```

Comment:
we already get the clc runner ip via the downward api https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L474

```suggestion
	podIP := config.Datadog.GetString("clc_runner_host")
```